tests/nested/manual/core20-install-device-file-install-via-hook-hack/snap-yaml-extras.yaml

```diff
@@ -18,3 +20,11 @@ plugs:
     interface: system-files
     write:
       - /run/mnt/ubuntu-data/system-data/_writable_defaults/etc/udev/rules.d/09-my-custom-udev.rules
+  systemd-journald-conf:
+    interface: system-files
+    write:
+      - /run/mnt/ubuntu-data/system-data/_writable_defaults/etc/systemd/journald.conf
```

Comment:
What if the they mess up journald.conf? Wold it break systemd-journal completely, such that it's possible end up in a state where logging does not work at all?

Perhaps should rather provide an example where system files allows access to `/etc/systemd/journald.conf.d/10-extra.conf`? I think we'd most likely need the core snaps to provide `/etc/systemd/journald.conf.d` directory, as IIRC system files interface only allows writing to the actual path and below, but not the prefix.